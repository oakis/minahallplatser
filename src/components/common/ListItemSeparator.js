import React from 'react';
import {View} from 'react-native';
import {colors} from '@style';

export const ListItemSeparator = () => {
  return (
    <View
      style={{
        borderBottomWidth: 1,
        borderBottomColor: colors.darkgrey,
      }}
    />
  );
};
