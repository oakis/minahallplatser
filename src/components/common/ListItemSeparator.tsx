import React from 'react';
import {View} from 'react-native';
import {colors} from '@style';

export const ListItemSeparator = (): JSX.Element => {
  return (
    <View
      style={{
        borderBottomWidth: 1,
        borderBottomColor: colors.darkgrey,
      }}
    />
  );
};
