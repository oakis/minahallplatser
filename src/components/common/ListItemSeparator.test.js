import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { ListItemSeparator } from './ListItemSeparator';

it('should match snapshot', () => {
    const wrapper = shallow(<ListItemSeparator />);
    expect(toJson(wrapper)).toMatchSnapshot();
});
