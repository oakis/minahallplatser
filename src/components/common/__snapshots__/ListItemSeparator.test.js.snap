// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#dedede",
      "borderBottomWidth": 1,
    }
  }
/>
`;
